.claude/skills/triage-leaked-infra/SKILL.md

```
@@ -0,0 +1,157 @@
+
+### 12. IAM roles (by infraID prefix)
+```bash
+aws iam list-roles --query "Roles[?starts_with(RoleName, '<infraID>')].[RoleName,CreateDate]" --output text
+```
+Report any IAM roles whose name starts with the infraID. These are orphaned OIDC-type roles (cloud-controller, ebs-csi, ingress, etc.) that belong to this infra set.
+
+## Output
+
+Present the report in this format:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)